build-support/bin/ci.py

```diff
@@ -242,6 +260,13 @@ def _use_remote_execution(oauth_token_path: str) -> List[str]:
     ]
 
 
+def _use_remote_cache(oauth_token_path: str) -> List[str]:
```

Comment:
Btw, you can delete the remote_execution code. It's stale and should have probably been deleted earlier.